inc/dashboard/provider.class.php

```diff
@@ -413,6 +416,317 @@ static function nbTicketsGeneric(
+         $data['series'][1]['data'][] = $resolveLate[$name];
+         $data['series'][2]['data'][] = $ownLate[$name];
+         $data['series'][3]['data'][] = $onTime[$name];
+      }
+
+      if (!isset($data['series'][0]['data'])) {
+         $data['series'][0]['data'] = [];
+         $data['series'][1]['data'] = [];
+         $data['series'][2]['data'] = [];
+         $data['series'][3]['data'] = [];
+         $data['labels'] = [];
+         $data['nodata'] = true;
+      }
+
+      return [
+         'label' => __('Tickets by SLA status and by technician'),
+         'truncate_boundary' => 'ending',
+         'data' => $data,
+         'icon' => $params['icon'],
+      ];
+   }
+
+   public static function nbTicketsByAgreementStatusAndTechnicianGroup(
+      string $case = "",
+      array $params = []
+   ):array {
+      $DBread = DBConnection::getReadConnection();
+
+      $default_params = [
+         'label'         => "",
+         'icon'          => 'far fa-clock',
+         'apply_filters' => [],
+      ];
+      $params = array_merge($default_params, $params);
+
+      $query_criteria  = [];
+      $search_criteria = [];
+
+      $laTable = Sla::getTable();
+      $laFk    = Sla::getForeignKeyField();
+      if ($case == 'Ola') {
+         $laTable = Ola::getTable();
+         $laFk    = Ola::getForeignKeyField();
+      }
+      $table = Ticket::getTable();
+      $laFk .=  '_ttr';
+      $ticketGroupTable = Group_Ticket::getTable();
+      $groupTable = Group::getTable();
+
+      $ownExceeded = Ticket::generateSLAOLAComputation('time_to_own', $table);
+      $resolveExceeded = Ticket::generateSLAOLAComputation('time_to_resolve', $table);
+      $slaState = "IF ($ownExceeded AND $resolveExceeded, 3, IF ($resolveExceeded, 2, IF ($ownExceeded, 1, 0)))";
+
+      $query_criteria = [
+         'COUNT' => 'cpt',
+         'SELECT'    => [
+            "$groupTable.name",
+            new QueryExpression("$slaState as `sla_state`"),
+         ],
+         'FROM'   => $table,
+         'INNER JOIN' => [
+            "$ticketGroupTable as gl" => [
+               'FKEY' => [
+                  'gl' => 'tickets_id',
+                  $table => 'id',
+                  [
+                     'AND' => [
+                        "gl.type" => Ticket_User::ASSIGN
+                     ]
+                  ]
+               ],
+            ],
+            $groupTable => [
+               'FKEY' => [
+                  $groupTable => 'id',
+                  'gl' => 'groups_id',
+               ],
+            ],
+         ],
+         'WHERE'  => [
+            "$table.is_deleted" => 0,
+         ] + getEntitiesRestrictCriteria($table),
+         'GROUPBY' => [
+            'sla_state',
+            "$groupTable.id",
+         ],
+         'ORDER' => [
+            'sla_state DESC',
+            'cpt DESC',
+            "$groupTable.id",
+         ]
+      ];
+
+      unset($params['apply_filters']['user_tech']);
```

Comment:
this card is not compatible with user_tech filter. To prevent false results, it is necessary to not apply the filter.

I did not find a proper way to do this. Comments welcome